src/todo-config.ts

```diff
@@ -14,64 +14,100 @@ import { TodoConfig } from './types';
  *     "daysToDecay": {
  *       "warn": 5,
  *       "error": 10
+ *     },
+ *     "daysToDecayByRule": {
+ *       "no-bare-strings": { "warn": 10, "error": 20 }
  *     }
  *   }
  * }
  * ```
  *
+ * A .lint-todorc.js file
+ *
+ * @example
+ * ```js
+ * module.exports = {
+ *   "daysToDecay": {
+ *     "warn": 5,
+ *     "error": 10
+ *   },
+ *   "daysToDecayByRule": {
+ *     "no-bare-strings": { "warn": 10, "error": 20 }
+ *   }
+ * }
```

Comment:
```suggestion
 * module.exports = {
 *.   "ember-template-lint": {
 *       "daysToDecay": {
 *         "warn": 5,
 *         "error": 10
 *       },
 *       "daysToDecayByRule": {
 *         "no-bare-strings": { "warn": 10, "error": 20 }
 *       }
 *.   }
 * }
```

Since we decided to nest by engine.